tests/app/template_cli_test.py

```diff
@@ -10,7 +10,7 @@
 def voila_args_extra():
     path_test_template = os.path.abspath(os.path.join(BASE_DIR, '../test_template/share/jupyter/voila/templates/test_template/'))
     path_default = os.path.abspath(os.path.join(BASE_DIR, '../../share/jupyter/voila/templates/default'))
-    return ['--template=None', '--VoilaTest.template_paths=[%r, %r]' % (path_test_template, path_default), '--VoilaExecutor.timeout=240']
+    return ['--template=None', f'--VoilaTest.template_paths=[{path_test_template, path_default}, {'--VoilaExecutor.timeout=240'}]'
```

Comment:
```suggestion
    return ['--template=None', f'--VoilaTest.template_paths=[{path_test_template}, {path_default}]', '--VoilaExecutor.timeout=240']
```

Reply:
Looks like the reason the tests are failing: https://github.com/voila-dashboards/voila/pull/419/checks?check_run_id=870226886